### DISCUSSION TITLE
How to handle large files?

### DISCUSSION THREAD

**Discussion by jS5t3r** [alshedivat/al-folio#2824] [Q&A]
My [script](https://github.com/lorenz-peter/lorenz-peter.github.io/blob/master/papers/fetch_papers.py) generates an json file which will automatically pushed to the repository. 

Well this file will grow over the time and might be too large for a github repository at some time. How to handle this?

**Comment by george-gca** [comment]
Check discussion #2242.

**Reply by jS5t3r** [reply]
Thanks for the answer. 

git lfs is known for me and for short term a solution.  Atm, I can push this json file without need of lfs. 
The maximum supported file size is 5GB which is fine for me and can work for some year.

However, I was thinking more about having something like gdrive or something for free.

**Reply by george-gca** [reply]
Idk how Google handles this, if you can share the file publicly and then use the public url to fetch the file. Maybe you can find a different hosting source for your file?

Another solution is to delete older less important papers. It would be my suggestion.
